server/src/dotConfig.mjs

```diff
@@ -58,6 +58,30 @@ function loadDotConfig() {
   return dotConfig;
 }
 
+/* 
+* @function:findFileCreationAndModificationTime
+* @Description: To get creation and modification time of files
+@param {String} creationDateFileName - Name of file whose creation time needs to be retrieved in seconds
+@param {String} modificationDateFileName - Name of file whose modification time needs to be retrieved in seconds
+* @Return: Array containing creation and modification time in seconds ex: [1687860231, 1687860231]
+*/
+
+function findFileCreationAndModificationTime(creationDateFileName, modificationDateFileName) {
```

Comment:
What you've written is fine and presumably works, but doing both in one function feels weird and makes the function less 'atomic.' I suggest splitting this into two separate functions, getCreationDate(fileName) and getModificationDate(fileName). Not a super high priority, but favoring smaller, focused functions is usually a good idea.

Reply:
Addressed